test/libsolidity/semanticTests/externalSource/_relative_imports/dir/B/b.sol

```diff
@@ -0,0 +1,11 @@
+import "../../c.sol";
```

Comment:
If this import works then there's something wrong. `c.sol` is one directory up from `b.sol` not two.

```suggestion
import "../c.sol";
```